plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -94,7 +94,8 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_PALTEX:
 			return "When checked 4/8 bits texture will be send to the GPU with a palette. GPU will be in charge of the conversion.\n\n"
 				"When unchecked the CPU will convert directly the texture to 32 bits.\n\n"
-				"It is basically a trade-off between GPU/CPU.";
+				"It is basically a trade-off between GPU/CPU.\n"
+				"For example you have a CPU that meets minimum of STR 1600 but not 2000 and a strong GPU GTX 1060 or stronger.";
```

Comment:
Old one was fine.

Reply:
Yeah perhaps shouldve been worded better, i think there is potential to improve it.